src/pages/SelectDate/SelectDatePage.tsx

```diff
@@ -9,24 +13,38 @@ import DateReservationCard from '@/shared/components/DateReservationCard/DateRes
 
 function SelectDatePage() {
   const [isSelected, setIsSelected] = useState(false);
-  const performanceData = {
-    performanceTitle: '연극제목',
-    performanceLocation: '공연장 위치',
-    performanceImage:
-      'https://tkfile.yes24.com/upload2/PerfBlog/202502/20250206/20250206-52586.jpg',
-    performanceGenre: '장르',
-    runningTime: '시간',
-    ageLimit: 18,
-  };
-  const performanceTime = '2025년 4월 1일';
-  const authors = '작가 이름';
+  const [performanceData, setPerformanceData] = useState<Performance | null>(
+    null
+  );
+  const [seatTypes, setSeatTypes] = useState<SeatGrades[] | null>(null);
+  const [authors, setAuthors] = useState('');
+  const [time, setTime] = useState('');
+  // API 연동
+  const { data, isLoading, isError } = useQuery<DatePageResponse>({
+    queryKey: [QUERY_KEY.HOME],
+    queryFn: getDateData,
+    staleTime: 1000 * 60 * 5,
+  });
```

Comment:
```suggestion
  const { data, isLoading, isError } = useDateData(); 
```

이미 훅 정의 해두셨으니까 가져오면 됩니당